sensor/kubernetes/selector/selector.go

```diff
@@ -1,50 +1,55 @@
-package resources
+package selector
 
 import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
-// labelWithLen is label.Labels with added Len() function
-type labelsWithLen interface {
+// LabelsWithLen is label.Labels with added Len() function
+type LabelsWithLen interface {
 	Has(label string) (exists bool)
 	Get(label string) (value string)
 	Len() uint
 }
 
-// selector is a restricted version of selectorWrap
-type selector interface {
-	Matches(labelsWithLen) bool
+// Selector is a restricted version of Wrap
+type Selector interface {
+	Matches(LabelsWithLen) bool
 }
 
 type labelWithLenImpl struct {
 	labels map[string]string
 }
 
+// Has label
 func (l labelWithLenImpl) Has(label string) bool {
 	_, exists := l.labels[label]
 	return exists
 }
 
+// Get label
 func (l labelWithLenImpl) Get(label string) string {
 	return l.labels[label]
 }
 
+// Len returns length of labels
 func (l labelWithLenImpl) Len() uint {
 	return uint(len(l.labels))
 }
 
-func createLabelsWithLen(labels map[string]string) labelWithLenImpl {
+// CreateLabelsWithLen create labels wrapper that respects the LabelsWithLen interface
+func CreateLabelsWithLen(labels map[string]string) LabelsWithLen {
 	return labelWithLenImpl{labels}
 }
 
-// selectorWrap holds a selector and information allowing for additional checks before matching
-type selectorWrap struct {
+// Wrap holds a selector and information allowing for additional checks before matching
+type Wrap struct {
```

Comment:
[nit] I don't think we need to export this.

Reply:
Done